core/image/BUILD.bazel

```diff
@@ -19,7 +19,7 @@ load("@rules_proto//proto:defs.bzl", "proto_library")
 go_library(
     name = "go_default_library",
     srcs = [
-        "astc.go",
+        "astc_format.go",
```

Comment:
Why the rename? They're all formats.

Reply:
the thing is because there is a package with same name, debugger gets confused where to jump when you try to debug. I changed so it can parse the package name correctly when the go runtime was calling "astc.init()". The other option is renaming the package. We can rename it do "astc_encoder" or something as well. As long as they have different name, both are fine for me.

Reply:
I have tried the rename the package, it requires a lot name change and it actually causes we call "astc_encoder.foo()" instead of "astc.foo()". So, if it is not very important to keep this astc.go I suggest it to be astc_format(or any other name) on a parallel work I am actually planning to do the same for etc1 & etc2 as well. I think being able to debug the package astc overcomes the a bit longer and unusual naming of a single source file.

Reply:
I don't know what you mean with debugger jumping, but don't do any renames - package or file - but file a bug to the debugger and will live with that for now.

Reply:
The thing is Go has a special condition for init functions. it's called by go runtime. And when the call comes to delve it comes as "core/image/astc.init()" and at this point delve does not if it's init function in astc.go or init function in astc package. Why does it has not come as "core/image/astc/astc.init()" or why delve try to show an init function "core/image/astc.go" when it does not exist. But in any case, it makes it hard to debug anything in the particular init function. Which how I got realise the issue.

I don't know how it is benefit us to keep the filename same. I can try to put a bug report for delve but I don't know when it would be fixed-or would it be fixed.

Simply, anytime we have a bug in compression or decompression, delve will be confused. And simply everytime it happens to me I'll rename the file, compile, solve and rename back to astc.go. Which I don't know if this is a necessary effort to just keep the filenames same as before.